### DISCUSSION TITLE
Hubs cloud deploy failure: Error creating budget: - the budget already exists. (w/ workaround)

### DISCUSSION THREAD

**Discussion by wmurphyrd** [mozilla/hubs#3407] [Hubs Cloud Questions] [answered]
Starting from a brand new AWS account an deploying Personal Hubs Cloud with domain recipe 2, I chose a database budget of $50, and found the deploy had failed and rolled back with this as the first error in the logs:

`Error creating budget: the budget already exists. (Service: AWSBudgets; Status Code: 400; Error Code: DuplicateRecordException;)`

I went to my budgets and found a budget with the same ID, and deleted it. Tried the deploy again with the same config, and received the same error. 

Third try, I deployed the hub with no database budget, and it succeeded. The budget created by the second attempt is still active in my account, so I think I'm good to go.

**Comment by robinkwilson** [comment, marked as answer]
Sounds like you are good to go, however, can you give me more information about your steps with deployment?

This could be a bug to track down. 
- Have you deployed an HC instance before in the same or a different region? -- Could be resource collision on the budget
- How many times did you deploy with rollbacks? -- could not be clearing correctly
- Did you ever delete a running HC stack? -- Perhaps something didn't clean up correctly

**Reply by robinkwilson** [reply]
Marking as answered for now to get it out of our pipeline, if you have more info, add the info then please mark as unanswered!
